### ISSUE TITLE
Destination JSON breaks when there is an escaped character at the end of one sentence

### ISSUE THREAD

**Issue by guillelopezgines** [while1618/i18n-auto-translation#2]
For example: `"tagged": "tagged \"{{ tags }}\"",`
```
{
  "general": {
    "404": {
      "continue_shopping": "Continue shopping",
      "open_menu": "Open menu",
      "skip_to_content": "Skip to content",
      "close_modal": "Close",
      "star_reviews_info": "{{ rating_value }} out of {{ rating_max }} stars",
      "tagged": "tagged \"{{ tags }}\"",
      "video_player": "Video player",
      "pagination": "Page {{ current_page }}"
    }
  }
}
```
Ends up in:
```
{
  "general": {
    "404": {
      "continue_shopping": "Continuer vos achats",
      "open_menu": "Ouvrir le menu",
      "skip_to_content": "Passer au contenu",
      "close_modal": "Fermer",
      "star_reviews_info": "{{ rating_value }} sur {{ rating_max }} étoiles",
      "tagged": "tagged \"{{ tags }}\"",
      "video_player": "Lecteur vidéo* |*Page {{ page_actuelle }}"
    }
  }
}
```
I don't know why, but if one line is removed, then it works correctly:
```
{
  "general": {
    "404": {
      "continue_shopping": "Continuer vos achats",
      "skip_to_content": "Passer au contenu",
      "close_modal": "Fermer",
      "star_reviews_info": "{{ rating_value }} sur {{ rating_max }} étoiles",
      "tagged": "tagged \"{{ tags }}\"",
      "video_player": "Lecteur vidéo",
      "pagination": "Page {{ page actuelle }}"
    }
  }
}
```

**Comment by while1618** [comment]
Fixed in v1.1.9

**Comment by guillelopezgines** [comment]
Thanks again for all these fixes. Really appreciate your work.  👍

When running my JSON I still see some translations that get messed up, ending in two translations on the same attribute.

Let me directly share my original JSON, so you can run the tests agains it:

```
{
  "general": {
    "404": {
      "title_html": "404 Page<br>not found",
      "subtitle": "The page you requested doesn't exist.",
      "continue_shopping_html": "Continue<br>shopping",
      "open_menu": "Open menu"
    },
    "accessibility": {
      "skip_to_content": "Skip to content",
      "close_modal": "Close",
      "video_player": "Video player",
      "star_reviews_info": "{{ rating_value }} out of {{ rating_max }} stars"
    },
    "meta": {
      "tags": "Tagged {{ tags }}",
      "page": "Page {{ page }}"
    },
    "search": {
      "title": "Search",
      "placeholder": "Search...",
      "no_results_for": "Your search for \"{{ terms }}\" did not yield any results.",
      "heading": {
        "one": "Search result",
        "other": "Search results"
      },
      "results_count": {
        "zero": "There are no results",
        "one": "{{ count }} result",
        "other": "{{ count }} results"
      },
      "results_with_count": {
        "one": "{{ count }} result for “{{ terms }}”:",
        "other": "{{ count }} results for “{{ terms }}”:"
      },
      "page": "Page",
      "article": "Blog"
    },
    "navigation":{
      "header_label": "MENU",
      "cancel_html": "or {{ link }}",
      "return_to_store": "Return to store"
    },
    "newsletter_form": {
      "newsletter_email": "Your email",
      "submit": "Subscribe",
      "confirmation": "Thanks for subscribing."
    },
    "page_title": {
      "tagged": "tagged \"{{ tags }}\"",
      "pagination": "Page {{ current_page }}"
    },
    "password_page": {
      "opening_soon": "Opening Soon",
      "login_form_heading": "Enter store using password:",
      "login_form_password_placeholder": "Your password",
      "login_form_submit": "Enter",
      "signup_form_heading": "Find out when we open:",
      "signup_form_email_placeholder": "Your email",
      "signup_form_submit": "Submit",
      "signup_form_success": "We will send you an email right before we open!",
      "social_heading": "Find us as well in:",
      "admin_link_html": "Are you the store owner? <a href=\"/admin\" class=\"text-link\">Log in here</a>",
      "password_link_html": "Enter with password",
      "back_link_html": "Back",
      "powered_by_shopify_html": "This shop will be powered by {{ shopify }}"
    },
    "country": {
      "dropdown_label": "Country/region"
    },
    "language": {
      "dropdown_label": "Language"
    }
  },
  "blogs": {
    "general": {
      "no_articles": "Sorry, there are currently no articles."
    },
    "article": {
      "meta_html": "Posted on {{ date }}",
      "read_entire_article_html": "read entire article",
      "next_article_html": "Newer Post &rarr;",
      "previous_article_html": "&larr; Older Post"
    },
    "comment_form": {
      "title": "Leave a comment",
      "name": "Name",
      "email": "Email",
      "message": "Message",
      "post": "Send",
      "moderated": "Please note, comments must be approved before they are published",
      "success_moderated_html": "Your comment was posted successfully.<br>We will publish it in a little while, as our blog is moderated.",
      "success": "Your comment was posted successfully! Thank you!"
    }
  },
  "collections": {
    "general": {
      "catalog": "Catalog",
      "collections": "Collections",
      "no_collections": "Sorry, there are currently no collections.",
      "no_products": "Sorry, there are currently no products available at this collection.",
      "arrow": "→",
      "filter_and_sort": "Filter and sort",
      "sort_button": "Sort",
      "filter_button": "Filter",
      "sort_by": "Sort by:",
      "filter_by": "Filter by:",
      "filters_selected": "{{ count }} selected",
      "max_price": "The highest price is {{ price }}",
      "product_count": "Showing {{ count }} of {{ total }} products",
      "from": "From",
      "to": "To",
      "reset": "Reset",
      "apply": "Apply",
      "clear": "Clear",
      "clear_all": "Clear all"
    },
    "tags": {
      "all": "all"
    }
  },
  "products": {
    "product": {
      "buy_html": "Buy<br>now",
      "on_sale_html": "Sale",
      "from_lowest_price_html": "From {{ lowest_price }}",
      "sold_out_html": "Sold<br>Out",
      "new_html": "New",
      "unavailable": "Unavailable",
      "quantity": "Quantity",
      "available_units_html": "<span class='units'>{{ units }}</span> available units",
      "add_to_cart": "Add to cart",
      "share_on": "Share on {{ platform}}",
      "view_in_space": "View in your space",
      "view_in_space_label": "View in your space, loads item in augmented reality window",
      "total_reviews": {
        "zero": "No reviews",
        "one": "{{ count }} review",
        "other": "{{ count }} reviews"
      }
    },
    "zoom": {
      "next": "Next (Right arrow key)",
      "prev": "Previous (Left arrow key)",
      "close": "Close (Esc)"
    }
  },
  "contact": {
    "form": {
      "section_title": "Contact",
      "title": "Get in touch",
      "name": "Name",
      "email": "Email",
      "phone": "Phone Number (optional)",
      "message": "Message",
      "send": "Send",
      "post_success": "Thanks for contacting us. We'll get back to you as soon as possible."
    }
  },
  "customer": {
    "login": {
      "section_title": "Sign in",
      "title": "Start new session",
      "email": "Email",
      "password": "Password",
      "sign_in_button": "Sign in",
      "forgot_password": "Forgot your password?",
      "register": "Create new account",
      "password_section_title": "Reset your password",
      "password_title": "You will receive an email with instructions",
      "password_action": "Send email",
      "password_cancel": "Cancel",
      "guest_section_title": "Don't want to register?",
      "guest_title": "Continue as a guest",
      "guest_action": "Skip register"
    },
    "reset_password": {
      "section_title": "Reset account password",
      "title": "Enter new password for {{ email }}",
      "password": "Password",
      "password_confirm": "Confirm password",
      "submit": "Reset password"
    },
    "register": {
      "section_title": "Create new account",
      "title": "Provide contact information",
      "first_name": "First Name",
      "last_name": "Last Name",
      "email": "Email",
      "password": "Password",
      "submit": "Create account",
      "login": "Already have an account?"
    },
    "account": {
      "section_title": "My account",
      "title": "Welcome back {{ customer_name }}",
      "add_address_html": "Add your address",
      "edit_addresses_html": "Edit addresses ({{ count }})"
    },
    "orders": {
      "order": "Order {{ id }}",
      "order_description_html": "<b>{{ date }}</b> — {{ order }} was placed and it's current status is {{ financial_status }} and {{ fulfillment_status }}",
      "empty": "There are no orders."
    },
    "order": {
      "section_title": "Order history",
      "date": "Placed on {{ date }}",
      "canceled": "Order Canceled on {{ date }}",
      "canceled_reason": "The reason was: {{ reason }}",
      "billing_address": "Billing Address",
      "payment_status": "Payment status",
      "shipping_address": "Shipping Address",
      "fulfillment_status": "Fulfillment status",
      "discount": "Discount",
      "price": "Price",
      "unit_price": "Unit price",
      "quantity": "Quantity",
      "subtotal": "Subtotal",
      "tax": "Tax",
      "total": "Total",
      "return_to_account": "Return to your account"
    },
    "activate_account": {
      "section_title": "Activate account",
      "title": "Set your password to activate your account",
      "password": "Password",
      "password_confirm": "Confirm password",
      "submit": "Activate account",
      "cancel": "Decline invitation"
    },
    "addresses": {
      "section_title": "Account details",
      "title": "Address book",
      "subtitle": "These are the addresses linked to your account",
      "empty": "Your Address Book is currently empty.",
      "default": "Default address",
      "add_new": "New address",
      "edit_address": "Edit address",
      "recipient": "Recipient",
      "first_name": "First name",
      "last_name": "Last name",
      "company": "Company",
      "address": "Address",
      "address1": "Address 1",
      "address2": "Address 2",
      "city": "City",
      "country": "Country",
      "province": "Province",
      "zip": "Postal/Zip code",
      "phone": "Phone",
      "phone_label": "Tel.",
      "set_default_label": "Set as default address?",
      "set_default_action": "Yes, please.",
      "add_address": "Add new address",
      "update": "Update address",
      "cancel": "Cancel",
      "edit": "Edit",
      "save": "Save address",
      "delete": "Delete",
      "delete_question": "Are you sure you wish to delete this address?",
      "return": "Return to your account"
    }
  },
  "sections": {
    "onboarding": {
      "featured_product_title": "Example Featured Product",
      "add_products_html": "Add your<br>products",
      "add_collection": "Collection name",
      "vendor": "Vendor name",
      "product_title": "Product title",
      "product_description": "Product description. Pair some text with a product to highlight features. This leather carrier is made to make your twin blanket a great camping companion. Camp blankets were first designed to meet the rigorous demands of the wild and untamed Pacific Northwest and were a favorite of the early sheepherders in the Northwest. Today, these warm and hardy blankets offer outstanding service and rugged styling that will provide years of warmth and comfort.",
      "blog_title": "Your blog's title",
      "post_title": "Your post's title",
      "post_excerpt": "Your store hasn’t published any blog posts yet. A blog can be used to talk about new product launches, tips, or other news you want to share with your customers. You can check out Shopify’s ecommerce blog for inspiration and advice for your own store and blog.",
      "page_title": "Your page's title",
      "page_excerpt_html": "This paragraph doesn’t currently include any page content.<br>Choose one of your pages using the sidebar.",
      "no_content_html": "This section doesn’t currently include any content.<br>Add content to this section using the sidebar."
    }
  },
  "cart": {
    "general": {
      "title_html": "Your Cart",
      "note": "Special instructions for seller:",
      "subtotal": "Subtotal",
      "taxes_and_shipping_policy_at_checkout_html": "Taxes and <a href=\"{{ link }}\">shipping</a> calculated at checkout.",
      "taxes_included_but_shipping_at_checkout": "Tax included and shipping calculated at checkout.",
      "taxes_included_and_shipping_policy_html": "Tax included. <a href=\"{{ link }}\">Shipping</a> calculated at checkout.",
      "taxes_and_shipping_at_checkout": "Shipping and taxes calculated at checkout.",
      "update_html": "Update cart",
      "checkout_html": "Check out",
      "empty": "Your cart is currently empty.",
      "continue_browsing_html": "Continue<br>browsing"
    }
  },
  "map": {
    "errors": {
      "address_error": "Error looking up that address",
      "address_no_results": "No results for that address",
      "address_query_limit_html": "You have exceeded the Google API usage limit. Consider upgrading to a <a href=\"https://developers.google.com/maps/premium/usage-limits\">Premium Plan</a>.",
      "auth_error_html": "There was a problem authenticating your Google Maps API Key."
    }
  },
  "store_availability":{
    "general":{
      "view_store_info": "View store information",
      "check_other_stores": "Check availability at other stores",
      "pick_up_available": "Pickup available",
      "pick_up_currently_unavailable": "Pickup currently unavailable",
      "pick_up_available_at_html": "Pickup available at <b>{{ location_name }}<\/b>",
      "pick_up_unavailable_at_html": "Pickup currently unavailable at <b>{{ location_name }}<\/b>"
    }
  },
  "gift_cards": {
    "issued":{
      "title": "Here's your {{ value }} gift card for {{ shop }}!",
      "subtext_html": "Here's your<br>gift card!",
      "amount_left_html": "{{ amount }} <small>left</small>",
      "redeem": "Use this code at checkout to redeem your gift card.",
      "shop_link_html": "Start<br>Shopping",
      "print": "Print Gift Card",
      "disabled": "This gift card has been disabled",
      "expired": "Expired on {{ expiry }}",
      "active": "Expires on {{ expiry }}",
      "add_apple_wallet": "Add To Apple Wallet"
    }
  },
  "layout": {
    "side_nav": {
      "account": "Your Account",
      "logout": "Log out",
      "signin": "Sign in",
      "back": "Back"
    },
    "footer": {
      "newsletter": "Sign up to our mailing list",
      "newsletter_placeholder": "Your email",
      "newsletter_submit": "join",
      "newsletter_success": "Thanks for subscribing.",
      "social": "Connect with us",
      "platform": "{{ name }} on {{ platform }}",
      "copyright_html": "&copy; Copyright {{ year }}, {{ shop_name }}."
    }
  },
  "date_formats": {
    "day_month_and_year": "%B %e, %Y",
    "month_and_year": "%B %Y",
    "date_and_time": "%B %e, %Y at %I:%M %P"
  }
}
```

Ends up being:
```
{
  "general": {
    "404": {
      "title_html": "404 Page<br>not found",
      "subtitle": "La page que vous avez demandée n'existe pas.",
      "continue_shopping_html": "Continuer<br>shopping",
      "open_menu": "Ouvrir le menu"
    },
    "accessibility": {
      "skip_to_content": "Passer au contenu",
      "close_modal": "Fermer",
      "video_player": "Lecteur vidéo",
      "star_reviews_info": " {{ rating_value }} sur {{ rating_max }} étoiles"
    },
    "meta": {
      "tags": "Tag {{ tags }}",
      "page": "Page {{ page }}"
    },
    "search": {
      "title": "Rechercher",
      "placeholder": "Rechercher...",
      "no_results_for": "Votre recherche pour \" {{ termes }}\" n'a donné aucun résultat.",
      "heading": {
        "one": "Résultat de la recherche",
        "other": "Résultats de la recherche"
      },
      "results_count": {
        "zero": "Aucun résultat",
        "one": "{{ count }} résultat",
        "other": "{{ count }} résultats"
      },
      "results_with_count": {
        "one": "{{ count }} résultat pour \"{{ termes }}\" :",
        "other": "{{ compte }} résultats pour \"{{ termes }}\" :"
      },
      "page": "Page",
      "article": "Blog"
    },
    "navigation": {
      "header_label": "MENU",
      "cancel_html": "ou {{ lien }}|* |Retourner au magasin",
      "return_to_store": "Votre email"
    },
    "newsletter_form": {
      "newsletter_email": "S'abonner",
      "submit": "Merci de vous être abonné.",
      "confirmation": "tagged \"{{ tags }}\""
    },
    "page_title": {
      "tagged": "Page {{ current_page }}",
      "pagination": "Ouverture prochainement"
    },
    "password_page": {
      "opening_soon": " Entrez dans le magasin en utilisant le mot de passe :",
      "login_form_heading": "Votre mot de passe",
      "login_form_password_placeholder": "Entrez",
      "login_form_submit": "Découvrez quand nous ouvrons :",
      "signup_form_heading": "Votre e-mail",
      "signup_form_email_placeholder": "Envoyer",
      "signup_form_submit": "Nous vous enverrons un e-mail juste avant l'ouverture !",
      "signup_form_success": "Trouver nous aussi dans :",
      "social_heading": "Êtes-vous le propriétaire du magasin ? <a href=\"/admin\" class=\"text-link\">Connectez-vous ici</a>",
      "admin_link_html": "Entrez avec mot de passe",
      "password_link_html": "Retour",
      "back_link_html": "Cette boutique sera propulsée par {{ shopify }}",
      "powered_by_shopify_html": "Pays/région"
    },
    "country": {
      "dropdown_label": " Langue"
    },
    "language": {
      "dropdown_label": "Désolé, il n'y a actuellement aucun article."
    }
  },
  "blogs": {
    "general": {
      "no_articles": "Publié le {{ date }}"
    },
    "article": {
      "meta_html": "lire l'article en entier",
      "read_entire_article_html": "Newer Post &rarr;",
      "next_article_html": "&larr; Message plus ancien",
      "previous_article_html": "Laisser un commentaire"
    },
    "comment_form": {
      "title": "Nom",
      "name": "E-mail",
      "email": "Message",
      "message": "Envoyer",
      "post": "Veuillez noter que les commentaires doivent être approuvés avant d'être publiés",
      "moderated": "Votre commentaire a été publié avec succès.<br> ;Nous le publierons dans quelques instants, car notre blog est modéré.",
      "success_moderated_html": "Votre commentaire a été posté avec succès ! Merci !",
      "success": "Catalogue"
    }
  },
  "collections": {
    "general": {
      "catalog": "Collections",
      "collections": "Désolé, il n'y a actuellement aucune collection.",
      "no_collections": "Désolé, il n'y a actuellement aucun produit disponible dans cette collection.",
      "no_products": "→",
      "arrow": "Filtrer et trier",
      "filter_and_sort": " Trier",
      "sort_button": "Filtrer",
      "filter_button": "Trier par :",
      "sort_by": "Filtrer par :",
      "filter_by": "{{ count }} sélectionné",
      "filters_selected": "Le prix le plus élevé est de {{ price }}",
      "max_price": "Afficher {{ count }} sur {{ total }} produits",
      "product_count": "De",
      "from": "À",
      "to": "Réinitialiser",
      "reset": "Appliquer",
      "apply": "Effacer",
      "clear": "Tout effacer",
      "clear_all": "tout"
    },
    "tags": {
      "all": "Acheter<br>maintenant"
    }
  },
  "products": {
    "product": {
      "buy_html": "Vente",
      "on_sale_html": "À partir de { {prix_le plus bas}}",
      "from_lowest_price_html": "Vendu<br>Épuisé",
      "sold_out_html": "Nouveau",
      "new_html": "Indisponible",
      "unavailable": "Quantité",
      "quantity": "<span class='units'>{{ units }}</span> unités disponibles",
      "available_units_html": "Ajouter au panier",
      "add_to_cart": "Partager sur {{ platform}}",
      "share_on": "Voir dans votre espace",
      "view_in_space": "Voir dans votre espace, charge l'article dans la fenêtre de réalité augmentée",
      "view_in_space_label": "Aucun avis",
      "total_reviews": {
        "zero": "{{ compte }} avis",
        "one": "{{ compter }} avis",
        "other": "Suivant (Flèche droite)"
      }
    },
    "zoom": {
      "next": "Précédent (Flèche gauche)",
      "prev": "Fermer (Échap)",
      "close": "Contact"
    }
  },
  "contact": {
    "form": {
      "section_title": "Nous contacter",
      "title": " Nom",
      "name": "Email",
      "email": "Numéro de téléphone (facultatif)",
      "phone": "Message",
      "message": "Envoyer",
      "send": "Merci de nous avoir contactés. Nous vous répondrons dans les plus brefs délais.",
      "post_success": "Connexion"
    }
  },
  "customer": {
    "login": {
      "section_title": "Démarrer une nouvelle session",
      "title": "E-mail",
      "email": "Mot de passe",
      "password": "Connexion",
      "sign_in_button": "Mot de passe oublié ?",
      "forgot_password": "Créer un nouveau compte| *|Réinitialiser votre mot de passe",
      "register": "Vous recevrez un e-mail avec des instructions",
      "password_section_title": "Envoyer un e-mail",
      "password_title": "Annuler",
      "password_action": "Vous ne voulez pas vous inscrire ?",
      "password_cancel": "Continuer en tant qu'invité",
      "guest_section_title": "Ignorer l'enregistrement",
      "guest_title": "Réinitialiser mot de passe du compte",
      "guest_action": "Saisir un nouveau mot de passe pour {{ email }}"
    },
    "reset_password": {
      "section_title": "Mot de passe",
      "title": "Confirmer le mot de passe",
      "password": "Réinitialiser le mot de passe",
      "password_confirm": "Créer un nouveau compte",
      "submit": "Fournir les informations de contact"
    },
    "register": {
      "section_title": "Prénom",
      "title": "Nom ",
      "first_name": "Email",
      "last_name": "Mot de passe",
      "email": "Créer un compte",
      "password": "Vous avez déjà un compte ?",
      "submit": "Mon compte",
      "login": "Bienvenue {{ nom_client }}"
    },
    "account": {
      "section_title": "Ajouter votre adresse",
      "title": "Modifier les adresses ({{ compte }})",
      "add_address_html": "Commande {{ id }}",
      "edit_addresses_html": "<b>{{ date }}</b> — {{ commande }} a été passée et son statut actuel est {{ financial_status }} et {{ Fulfulment_status }}"
    },
    "orders": {
      "order": "Aucune commande.",
      "order_description_html": "Historique des commandes",
      "empty": "Passé le {{ date }}| *|Commande annulée le {{ date }}"
    },
    "order": {
      "section_title": "La raison était : {{ Reason }}",
      "date": "Adresse de facturation",
      "canceled": "État du paiement",
      "canceled_reason": "Adresse de livraison",
      "billing_address": "État d'exécution",
      "payment_status": "Remise",
      "shipping_address": "Prix ",
      "fulfillment_status": "Prix unitaire",
      "discount": "Quantité",
      "price": "Sous-total",
      "unit_price": "Taxe",
      "quantity": "Total",
      "subtotal": "Revenir à votre compte",
      "tax": "Activer le compte",
      "total": "Définissez votre mot de passe pour activer votre compte",
      "return_to_account": "Mot de passe"
    },
    "activate_account": {
      "section_title": " Confirmer le mot de passe",
      "title": "Activer le compte",
      "password": "Refuser l'invitation",
      "password_confirm": "Détails du compte",
      "submit": "Carnet d'adresses",
      "cancel": "Voici les adresses liées à votre compte"
    },
    "addresses": {
      "section_title": "Votre carnet d'adresses est actuellement vide.",
      "title": "Adresse par défaut|* |Nouvelle adresse",
      "subtitle": "Modifier l'adresse",
      "empty": "Destinataire",
      "default": "Prénom",
      "add_new": "Nom",
      "edit_address": "Société",
      "recipient": "Adresse",
      "first_name": "Adresse 1",
      "last_name": "Adresse 2",
      "company": "Ville",
      "address": "Pays|* |Province",
      "address1": "Code postal/code postal",
      "address2": "Téléphone",
      "city": "Tél.",
      "country": "Définir comme adresse par défaut ?",
      "province": "Oui, s'il vous plaît.",
      "zip": "Ajouter une nouvelle adresse",
      "phone": "Mettre à jour l'adresse",
      "phone_label": "Annuler|* |Modifier",
      "set_default_label": "Enregistrer l'adresse",
      "set_default_action": "Supprimer",
      "add_address": "Êtes-vous sûr de vouloir supprimer cette adresse ?",
      "update": "Retournez à votre compte",
      "cancel": "Exemple de produit en vedette",
      "edit": "Ajoutez vos produits<br>",
      "save": "Nom de la collection",
      "delete": "Nom du fournisseur",
      "delete_question": "Titre du produit",
      "return": "Description du produit. Associez du texte à un produit pour mettre en évidence les fonctionnalités. Ce sac de transport en cuir est conçu pour faire de votre couverture jumelle un excellent compagnon de camping. Les couvertures de camp ont d'abord été conçues pour répondre aux exigences rigoureuses du nord-ouest sauvage et indompté du Pacifique et étaient les préférées des premiers bergers du nord-ouest. Aujourd'hui, ces couvertures chaudes et résistantes offrent un service exceptionnel et un style robuste qui fourniront des années de chaleur et de confort."
    }
  },
  "sections": {
    "onboarding": {
      "featured_product_title": "Titre de votre blog",
      "add_products_html": "Titre de votre article",
      "add_collection": "Votre boutique n'a publié aucun article de blog encore. Un blog peut être utilisé pour parler des lancements de nouveaux produits, des conseils ou d'autres nouvelles que vous souhaitez partager avec vos clients. Vous pouvez consulter le blog de commerce électronique de Shopify pour trouver de l'inspiration et des conseils pour votre propre boutique et blog.",
      "vendor": "Titre de votre page",
      "product_title": "Ce paragraphe n'inclut actuellement aucun contenu de page.<br>Choisissez l'une de vos pages en utilisant la barre latérale.",
      "product_description": "Cette section n'inclut actuellement aucun contenu.<br>Ajoutez du contenu à cette section à l'aide de la barre latérale.",
      "blog_title": "Votre panier",
      "post_title": "Instructions spéciales pour le vendeur :",
      "post_excerpt": "Sous-total",
      "page_title": "Taxes et <a href=\"{{ lien }}\">livraison</a> calculé à la caisse.",
      "page_excerpt_html": "Taxes incluses et frais de port calculés à la caisse.",
      "no_content_html": "Taxes incluses. <a href=\"{{ lien }}\">Livraison</a> calculé à la caisse."
    }
  },
  "cart": {
    "general": {
      "title_html": "Livraison et taxes calculées à la caisse.",
      "note": "Mettre à jour le panier",
      "subtotal": "Passer à la caisse",
      "taxes_and_shipping_policy_at_checkout_html": "Votre panier est actuellement vide.",
      "taxes_included_but_shipping_at_checkout": "Continuer<br>naviguer",
      "taxes_included_and_shipping_policy_html": "Erreur lors de la recherche de cette adresse| *|Aucun résultat pour cette adresse",
      "taxes_and_shipping_at_checkout": "Vous avez dépassé la limite d'utilisation de l'API Google. Envisagez de passer à un <a href=\"https://developers.google.com/maps/premium/usage-limits\">forfait Premium</a>.",
      "update_html": "Un problème est survenu lors de l'authentification de votre API Google Maps Clé.",
      "checkout_html": "Afficher les informations sur le magasin",
      "empty": "Vérifier la disponibilité dans d'autres magasins",
      "continue_browsing_html": "Retrait disponible"
    }
  },
  "map": {
    "errors": {
      "address_error": "Retrait actuellement indisponible",
      "address_no_results": "Retrait disponible à <b>{{ location_name }}</b>",
      "address_query_limit_html": " Ramassage actuellement indisponible à <b>{{ location_name }}</b>",
      "auth_error_html": "Voici votre carte-cadeau de {{ value }} pour {{ shop }} !"
    }
  },
  "store_availability": {
    "general": {
      "view_store_info": "Voici votre<br>cadeau carte !",
      "check_other_stores": "{{ montant }} <petit>gauche</petit>",
      "pick_up_available": "Utilisez ce code lors du paiement pour utiliser votre carte-cadeau.",
      "pick_up_currently_unavailable": "Commencer<br>Shopping",
      "pick_up_available_at_html": "Imprimer la carte-cadeau",
      "pick_up_unavailable_at_html": " Cette carte-cadeau a été désactivée"
    }
  },
  "gift_cards": {
    "issued": {
      "title": "Expire le {{ expiration }}",
      "subtext_html": "Expire le {{ expiration }}",
      "amount_left_html": "Ajouter à Apple Wallet",
      "redeem": "Votre compte",
      "shop_link_html": "Déconnectez-vous",
      "print": "Connexion|* |Retour",
      "disabled": "Inscrivez-vous à notre liste de diffusion",
      "expired": "Votre e-mail",
      "active": "rejoindre",
      "add_apple_wallet": "Merci de vous être abonné."
    }
  },
  "layout": {
    "side_nav": {
      "account": "Rejoignez-nous",
      "logout": "{{ nom }} sur {{ plateforme }}",
      "signin": " &copier; Copyright {{ année }}, {{ shop_name }}.",
      "back": "%B %e, %Y"
    },
    "footer": {
      "newsletter": "%B %Y",
      "newsletter_placeholder": "%B %e, %Y à %I :%M %P"
    }
  },
  "date_formats": {}
}
```
Notice the:
- `"cancel_html": "ou {{ lien }}|* |Retourner au magasin",`
- `"title": "Adresse par défaut|* |Nouvelle adresse",`
- `"address": "Pays|* |Province",`
- `"phone_label": "Annuler|* |Modifier",`
- `"print": "Connexion|* |Retour",`

**Comment by while1618** [comment]
Hey @guillelopezgines, I just wanted to check on your second comment, but I can't find it anymore. Is it working as expected now, and you just didn't update the library, so you deleted that comment, or is it something else?

**Comment by guillelopezgines** [comment]
Hey, after the update I was still seeing some errors as my JSON includes some i18next syntax. I end up finding another service that fits better for my needs: https://github.com/fkirc/attranslate. Thanks though for all your hard work fixing all these issues.

**Comment by while1618** [comment]
Hey @guillelopezgines, I saw that library in the middle of my work, but I still wanted to finish mine. I think that their library is a better one, because it has much more features, but I think mine is easier to use, in my opinion it has a better CLI. Of course, if you need only to translate JSON files, for other cases it's better to go with their library. If you want to continue with mine, feel free to open any new issue if you find it, if not, I'm glad I could help you at least with something.

**Comment by guillelopezgines** [comment]
So kind of you! I think I am right with that other library, it's working pretty well, but thank you very much for your offering. I really appreciate.

**Comment by while1618** [comment]
That weird space in delimiter should no longer be a problem. It should be fixed in v1.1.11.
